modules/ROOT/pages/hybrid-manage-props.adoc

```diff
@@ -10,11 +10,29 @@ image:logo-server-active.png[link="/runtime-manager/deployment-strategies#anypoi
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
 
+
 In an on-premises (hybrid or Anypoint Platform PCE) deployment, you can add properties to an application using the `mule-app.properties` file or Anypoint Runtime Manager.
 
-== Add Application Properties with the mule-app.properties file
+For example, you can define application-specific properties in a properties file, then later add environment-specific properties from Runtime Manager during deployment.
+
+Specifying properties in Runtime Manager enables you to define properties in one place rather than having multiple application property files for a single application in multiple environments.
+
+After deploying an application, you can modify application properties or create new ones from the application *Settings* page.
+The values of properties defined on this tab override any application properties defined in the `mule-app.properties` or other properties of Mule.
+
+If you change properties after an application is deployed, you must restart the application and any server group or cluster where the application is deployed.
+
+The following security considerations apply to properties:
+
+* Values sent between Runtime Manager and Mule runtime engine are encrypted over HTTPS.
+* Values stored in the Runtime Manager internal database are encrypted.
+* By default, Runtime Manager does not display the values of the properties.
```

Comment:
What does this mean? If the user has access to the settings page, as stated below, the UI will receive the properties as plain

Reply:
Yeah, that was taken from this topic:
https://docs.mulesoft.com/runtime-manager/deploying-to-your-own-servers#properties-tab

I've changed it to be clearer...hopefully.

Reply:
Looks good to me!